frontend/src/main/java/com/google/tripmeout/frontend/storage/PersistentStorageBindingModule.java

```diff
@@ -0,0 +1,11 @@
+package com.google.tripmeout.frontend.storage;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Singleton;
+
+public class PersistentStorageBindingModule extends AbstractModule {
+  @Override
+  public void configure() {
+    bind(TripStorage.class).to(PersistentTripStorage.class).in(Singleton.class);
```

Comment:
You'll also need to bind any dependencies of `PersistentTripStorage`. In this case, that's just `DatastoreService`.

Since that's accessed through an external factory method, you'll want to expose that through a [`@Provides` method](https://github.com/google/guice/wiki/ProvidesMethods).